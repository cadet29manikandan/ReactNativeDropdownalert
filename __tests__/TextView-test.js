import React from 'react';
import TextView from '../TextView';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';

test('renders TextView with text', () => {
  const wrapper = shallow(<TextView text={'Hello World'} />);
  const tree = toJson(wrapper);
  expect(tree).toMatchSnapshot();
});
test('renders TextView without text', () => {
  const wrapper = shallow(<TextView text={''} />);
  const tree = toJson(wrapper);
  expect(tree).toEqual('');
});
