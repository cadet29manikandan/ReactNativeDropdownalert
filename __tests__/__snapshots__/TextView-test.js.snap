// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TextView with text 1`] = `
<Text
  numberOfLines={1}
  style={
    Object {
      "backgroundColor": "transparent",
      "color": "white",
      "fontSize": 16,
      "fontWeight": "normal",
      "textAlign": "left",
    }
  }
  text="Hello World"
>
  Hello World
</Text>
`;
